srcpkgs/opensnitch/patches/musl-int.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/daemon/netfilter/queue.h b/daemon/netfilter/queue.h
+index 64c3ea7c0c..2ced3bb39e 100644
+--- a/daemon/netfilter/queue.h
++++ b/daemon/netfilter/queue.h
+@@ -14,6 +14,11 @@
+ #include <linux/netfilter.h>
+ #include <libnetfilter_queue/libnetfilter_queue.h>
+
++typedef unsigned int uint;
++typedef uint8_t u_int8_t;
++typedef uint16_t u_int16_t;
++typedef uint32_t u_int32_t;
```

Comment:
Please try upstreaming usage of proper types

Just `uintX_t` is enough and `unsigned` instead of `uint`.

Reply:
> Please try upstreaming usage of proper types

To clarify, do you mean that I should try upstreaming the patch after I fix it, or should I try altering the rest of the header to use the full type definitions and upstream that?

Reply:
Try upstreaming a change in upstream code to use `uintX_t` directly instead of `u_intX_t`, since the latter is silly and nonstandard. Same for `unsigned` instead of `uint`.

Reply:
The typedefs shouldn't be necessary, because the types shouldn't be used

Reply:
Before I submit my changes upstream, can you take a look at the changes I made in the patch and see if they look good?

Reply:
My changes have been accepted upstream and will likely be included in the next release.